### PR TITLE
doc: add minutes for meeting 29 Nov 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2017-11-29.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* Enable explicit `.m.js` intent for ESM [#16170](https://github.com/nodejs/node/pull/16170)
+  * Bradley, that thread contains the thought that we don.t need to import non-esm modules.
```
Comment (maclover7) [inline meetings/2017-11-29.md]: `don.t` --> `don't`

File: meetings/2017-11-29.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    There are a number so should likely be more generic to handle others like wasm,
+    json etc. 2 older formats that should be included in the discussion as well.
+  * Next action action, Myles to schedule working session which includes this and other
```
Comment (maclover7) [inline meetings/2017-11-29.md]: `Next action action` --> `Next action`

File: meetings/2017-11-29.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    topics that we don.t have consensus on.
```
Comment (maclover7) [inline meetings/2017-11-29.md]: `don.t` --> `don't`

File: meetings/2017-11-29.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  * Question was: If and when Buffer constructor is deprecated, might Buffer(num) be deprecated
+    before other forms of Buffer constructor? Or will they be deprecated all at once?
+  * Take it back to github for a week, need champion to better frame what the decision is we need
+    now and why.
+* timers: setInterval interval includes cb duration [#14815](https://github.com/nodejs/node/pull/14815)
+  * Tagged by Jeremiah
+  * Current set interval has callback that takes a significant amount of time, next interval will
+    be scheduled after the callback end.  PR changes to schedule so that it is based on 
+    when the previous one started.
+  * Light discussion in the TSE meeting.  New behaviour sounds right (needs to be
```
Comment (maclover7) [inline meetings/2017-11-29.md]: TSE --> TSC

File: meetings/2017-11-29.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+### nodejs/TSC
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * quick update was provided by champions
+
+* Register Node specific MIME types [#371](https://github.com/nodejs/TSC/issues/371)
+  * Bradley - suggesting we make the controller the Foundation as opposed to the TSC. 
+  * James - should be ok if we tie to the TSC.  Does not need to be legal entity.
+  * Bradley - they are asking for .standards. org.  That is what should be the Foundation
+  * James - don.t think it matters.
```
Comment (maclover7) [inline meetings/2017-11-29.md]: `don.'t` --> `don't`

File: meetings/2017-11-29.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+  * Bradley, leave TSC as change controller and standard org is identified as Foundation.
+  * Rod - rather have the TSC as both.
+  * We.ll move forward with it as the TSC.
```
Comment (maclover7) [inline meetings/2017-11-29.md]: `We.ll` --> `We'll`

File: meetings/2017-11-29.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Evan Lucas @evanlucas (TSC)
+* James M Snell @jasnell (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Michael Dawson @mhdawson (TSC)
+* MichaëZasso @targos (TSC)
+* Rod Vagg @rvagg (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* Bradley Meck @bmeck (Obserer)
```
Comment (maclover7) [inline meetings/2017-11-29.md]: bmeck is mentioned twice

**Comment by mhdawson** [comment]
@maclover7 thanks for the comments, @Trott thanks for the fixes :)

**Pull request merged**